docs/app/pages/Components/Input/Input.vue

```diff
@@ -20,6 +22,20 @@
       <code-example title="Input and Textarea" :component="examples['text-fields']" />
     </div>
 
+    <div class="page-container-section">
+      <h2>Field Variations</h2>
+
+      <p>Fields have variations of thre layouts: Default with floating labels, a floating labels with boxes and a boxed layout with inline labels. </p>
```

Comment:
Can you improve the description of the sections?

Reply:
Sorry, my English is not much good. if you give me the description as reply here, then i can update.

Reply:
Fields have three layout variations:  Default (with floating labels), solo (with inline labels and boxed (with floating labels).